numalogic/udfs/postprocess.py

```diff
@@ -197,7 +198,7 @@ def _per_feature_score(feat_names: list[str], scores: NDArray[float]) -> dict[st
     @classmethod
     def compute(
         cls, model: artifact_t, input_: NDArray[float], postproc_clf=None, **_
-    ) -> NDArray[float]:
+    ) -> tuple[NDArray[float], NDArray[float]]:
```

Comment:
I thought we said we donot want to return 2 things when working on this.

Reply:
Yes as discussed, will change it.